Source/Resources/Content/MSGraph/DeviceAppManagement/WindowsInformationProtectionPolicies/Protect Policy Managed Client Apps on Unmanaged Windows Devices.json

```diff
@@ -15,7 +15,7 @@
   ],
   "azureRightsManagementServicesAllowed": false,
   "daysWithoutContactBeforeUnenroll": 90,
-  "description": "Configures default behavior for policy managed client apps on unmanaged Windows devices.\n\nHardens security for policy managed apps and blocks egress of organization data.",
+  "description": "What is it: Configures default behavior for managed client apps on unmanaged Windows devices. Hardens security for managed apps and blocks egress of organization data. \n Simeon Notes: Protect your data on users' personal Windows devices. This policy allows data to flow between approved apps, but does not allow users to copy or save data outside of the app. Simeon's baseline applies this policy to all users except those in the Azure AD group, 'BUILTIN - Unrestricted Access From Unmanaged Devices And Untrusted Locations' \n Learn More: https://docs.microsoft.com/en-us/mem/intune/apps/app-protection-policy",
```

Comment:
**What does this do?**
Configures the default behavior for managed client apps on unmanaged Windows devices. The baseline allows data to flow between approved apps, but prohibits users from copying or saving data outside of the app except for users in the Azure AD group, 'Baseline - Unrestricted Access From Unmanaged Devices And Untrusted Locations'. 

@metamenos do we need to specify where/which apps are "approved"? (2nd sentence)

**Why should you use this?**
 If you want to further protect your data on users' personal Windows devices by blocking egress of organization data. 

**What is the end-user impact?**
 Windows device users cannot copy or save data outside of your managed client apps. 

**Learn more**
[How you can protect app data](https://docs.microsoft.com/en-us/mem/intune/apps/app-protection-policy)

Reply:
Yes agree with your suggestion

Reply:
do we need to specify where/which apps are "approved"? (2nd sentence)